src/zenodo_client/struct.py

```diff
@@ -121,6 +121,9 @@ class Metadata(BaseModel):
     license: Optional[str] = "CC0-1.0"
     publication_type: Optional[PublicationType] = None
     image_type: Optional[ImageType] = None
+    communities: Optional[list] = []
+    keywords: Optional[list] = []
```

Comment:
Let's improve the type annotation in the following ways:

1. Use backwards compatible `List`
2. More carefully annotate the type of list with `List[str]`
3. Don't use mutable defaults (https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)
   - Option 1 is to use `pydantic.Field(default_factory=list)`
   - Option 2 is to take advantage of the fact that you marked this as optional and use `None`

Reply:
see updates in https://github.com/cthoyt/zenodo-client/commit/dd69b7a646f818aa9c732a17e11046d01d9c97e6

Reply:
Thanks for the Python lesson on the side ! Makes sense.